### ISSUE TITLE
fake_stack_prof swallows forgetting to add stackprof to system

### ISSUE THREAD

**Issue by schneems** [scoutapp/scout_apm_ruby#47]
Lost about a half an hour of my day to:

```
/Users/richardschneeman/.gem/ruby/2.3.1/gems/scout_apm-2.0.0.pre/lib/scout_apm/utils/fake_stack_prof.rb
```
Turns out the system didn't have `stackprof` in the Gemfile but I didn't realize because I wasn't getting any errors when trying to directly use `StackProf.run`.

Maybe try to require it and output some warning code if it cannot be loaded:

```
begin
  require 'stackprof'
rescue LoadError
  warn "Could not load `stackprof` gem, make sure it's on your system. If your system cannot run stackprof disable this message with SCOUT_DISABLE_STACKPROF_WARNING=1 environment variable" unless ENV["SCOUT_DISABLE_STACKPROF_WARNING"]
end
```

**Comment by cschneid** [comment]
I'm sorry about hiding the error from you, I'll be releasing a new version of the gem tomorrow without the stackprof shim at all.

**Comment by cschneid** [comment]
This is fixed & released in `scout_apm` version `2.0.0.pre3` and `1.6.1`.

**Comment by schneems** [comment]
Word, thanks!
